src/main/java/com/thirdcc/webapp/service/UserService.java

```diff
@@ -372,6 +374,25 @@ public User registerFirebaseUser(String fullName, String uid, String email, Stri
             .collect(Collectors.toList());
     }
 
+    /**
+     * Gets a list of users that do not have assigned family.
+     *
+     * @return a list of all users without assigned family.
+     */
+    public List<UserDTO> getUsersWithFamilyCheck(Pageable pageable, boolean hasFamily){
+        List<UserDTO> userDTOList = userRepository.findAllByLoginNot(pageable, Constants.ANONYMOUS_USER).map(UserDTO::new).getContent();
```

Comment:
What is this ANONYMOUS_USER meaning for?
Why need this?

Reply:
Actually I was referring from a previous code sample.
But this has been resolved in 5940d2bd46afc1053fdcb4d95e313b57bdf44cde